src/pickers/InsureeStatusReasonPicker.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { useModulesManager, useTranslations, Autocomplete, useGraphqlQuery } from "@openimis/fe-core";
+import _debounce from "lodash/debounce";
+
+const InsureeStatusReasonPicker = (props) => {
+  const { onChange, readOnly, required, value, filterOptions, filterSelectedOptions, multiple, extraFragment, edited } =
+    props;
```

Comment:
Props can be destructed like this:
`
const InsureeStatusReasonPicker = ({
onChange,
readOnly,
required,
value, 
filterOptions,
filterSelectedOptions,
multiple,
extraFragment,
edited,
}) => {...}
`